cob_default_robot_config/cob4-2/behaviour.yaml

```diff
@@ -0,0 +1 @@
+service_ns: /behaviour
```

Comment:
I need it to know the name space of the trigger services , for example for the command_gui_buttons (it has to correspond with the component_name ).